utils/utils.go

```diff
@@ -0,0 +1,43 @@
+package utils
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+func GetOwnerRepoPull(ref, repo string) (string, string, int) {
+	// get pull request files
+	githubrefS := strings.Split(ref, "/")
+	prnum := githubrefS[2]
+	pull, _ := strconv.Atoi(prnum)
+
+	repoS := strings.Split(repo, "/")
+	owner := repoS[0]
+	repoName := repoS[1]
+
+	return owner, repoName, pull
+}
+
+func ValidateModuleSource(source string, approvedModules map[string]bool) (bool, error) {
+	for k, v := range approvedModules {
+		if strings.Contains(source, k) {
+			if v {
+				return true, nil
```

Comment:
Does this return mean that we return early and don't actually loop over all the approved modules?

Reply:
so the source file is run against a list of approved modules, if the source file is in that list it will return true